website/www/site/content/en/contribute/_index.md

```diff
@@ -73,20 +73,31 @@ detail.
 
 ### Prerequisites
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
+ - A GitHub account.
+ - A Linux, macOS, or Microsoft Windows development environment
+ - Java JDK 8 installed
```

Comment:
Is there an instruction here which we could link to here?
This usually requires a few steps.
Downloading and unpacking in an appropriate spot
Setting up a JAVA_HOME var
Verifying java -version is using java 8.

Reply:
The Java verification goes later described in the section **Development Setup**. 
Maybe these specific instructions could go in the Wiki  https://cwiki.apache.org/confluence/display/BEAM/Java+Tips